src/renderer/components/swap/Swap.utils.ts

```diff
@@ -15,11 +15,11 @@ import { AssetWithPrice } from '../../services/binance/types'
  *          some(false) - sourceAsset is RUNE
  */
 export const isRuneSwap = (sourceAsset: Asset, targetAsset: Asset) => {
-  if (isRuneAsset(targetAsset)) {
+  if (isRuneBnbAsset(targetAsset)) {
```

Comment:
dont we have to care about `RuneNative` ?

Reply:
Good point!